### ISSUE TITLE
The unit of BannerInstruction and InstructionList in InstructionView are different

### ISSUE THREAD

**Issue by JunDai** [mapbox/mapbox-navigation-android#3257]
### Steps to trigger behavior
The device locale is `en`
 1. set a `distanceFormatter` with `uniteType = METRIC ` and then startNavigation
```kotlin
val navigationOptions =
    MapboxNavigation.defaultNavigationOptionsBuilder(
        applicationContext,
        Mapbox.getAccessToken()
    ).distanceFormatter(
        MapboxDistanceFormatter.Builder()
            .withUnitType(METRIC)
            .build(applicationContext)
    )
        .build()
optionsBuilder.navigationOptions(navigationOptions)

navigationView.startNavigation(optionsBuilder.build())
```
 2. check InstructionList view

### Expected behavior
The BannerInstruction and InstructionList should use the same unit.

### Actual behavior
The BannerInstruction uses the **METRIC** and the InstructionList uses the **IMPERIAL**.

![image](https://user-images.githubusercontent.com/3918950/86163880-c387c500-bac5-11ea-8e35-b6d122ce36e3.png)

**Comment by JunDai** [comment]
we will see this issue as long as the `distanceFormatter's unitType in NavigationOptions` doesn't match `directionRoute's voiceUnits`.

**Comment by JunDai** [comment]
closed by #3270
